pushbullet

```diff
@@ -7,6 +7,8 @@ API_URL=https://api.pushbullet.com/v2
 PROGDIR="$(cd "$( dirname "$0" )" && pwd )"
 unset QUIET
 
+CURL="curl"
+
```

Comment:
Since there's already a $(which curl) check in an if statement near the beginning of the script to verify that curl is available, using just "curl" should be fine.